app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>All Recipes</h1>
```

Comment:
This file is not required by the user stories - you can build this out for practice afterwards, but I would not worry about anything not explicitly required in the user stories during these timed ICs.

Reply:
That having been said, this page is implemented correctly!